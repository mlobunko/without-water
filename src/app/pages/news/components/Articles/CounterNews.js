import React from 'react';
import PropTypes from 'prop-types';

import { StyledCounterNews } from './styles';

export const CounterNews = ({ total }) => (
  <StyledCounterNews>
    <span>Total News: {total}</span>
  </StyledCounterNews>
);

export default CounterNews;

CounterNews.propTypes = {
  total: PropTypes.number.isRequired
};
