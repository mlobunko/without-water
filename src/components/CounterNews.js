import React from "react";

export const CounterNews = ({ total }) => (
  <div className="counter-news">
    <span>Total News: {total}</span>
  </div>
);
