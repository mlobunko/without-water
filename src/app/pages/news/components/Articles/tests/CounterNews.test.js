import React from 'react';
import { shallow } from 'enzyme';

import { CounterNews } from '../CounterNews';

describe('render', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<CounterNews total={1} />);
    expect(wrapper).toMatchSnapshot();
  });
});
