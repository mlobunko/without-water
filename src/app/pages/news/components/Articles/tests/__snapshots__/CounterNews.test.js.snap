// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render renders correctly 1`] = `
<styled.div>
  <span>
    Total News: 
    1
  </span>
</styled.div>
`;
